scrapers/scrape_be_districts.py

```diff
@@ -6,20 +6,30 @@
 
 
 def fix_city(city):
+    # fix common abbreviations
+    city = re.sub(r' b\.\s?B\.', ' bei Bern', city)
+    city = re.sub(r' a\.\s?A\.', ' an der Aare', city)
+    city = re.sub(r' i\.\s?E\.', ' im Emmental', city)
+
+    # and handle a few special cases
     cities = {
-        'Biel': 'Biel / Bienne',
-        'Wohlen b. B.': 'Wohlen bei Bern',
-        'Muri-Gümligen': 'Muri bei Bern',
         'St-Imier': 'Saint-Imier',
-        'Büren a.A.': 'Büren an der Aare',
-        'Langnau i.E.': 'Langnau im Emmental',
-        'Oberhofen': 'Oberhofen am Thunersee',
-        'Erlenbach': 'Erlenbach im Simmental',
```

Comment:
What about Biel, Oberhofen and Erlenbach? I think
 those should be kept.

Reply:
And Muri-Gümligen, too

Reply:
No, this shouldn't be required anymore, unless for example multiple 'Erlenbach' would exist in the canton.
Sorry I forgot to mention this in the original comment, but on line 118+, this should now be handled by looking for the city name provided by the page in the available cities.
I'll re-add Muri-Gümligen, I removed this one by mistake...